controller/comments.go

```diff
@@ -60,8 +60,7 @@ func (c *CommentsController) Show(ctx *app.ShowCommentsContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		cmt, err := appl.Comments().Load(ctx, ctx.CommentID)
 		if err != nil {
-			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(ctx, goa.ErrUnauthorized(err.Error()))
-			return ctx.NotFound(jerrors)
+			return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
```

Comment:
This is technically not the same. 404 != 401. Unsure what the original intent here was

Reply:
What is the right error to return here?

Reply:
jsonapi.JSONErrorResponse(ctx, errors.NewNotFoundError(...))
// import "github.com/fabric8-services/fabric8-wit/errors"